lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -932,6 +967,29 @@ def get_webapp_configuration(self):
 
             return False
 
+    def start_webapp(self, action):
```

Comment:
this name is a bit misleading, because it can be also used to stop the application.

Reply:
maybe set_application_state would be better

Reply:
changed to change_webapp_state